upi/metal/main.tf

```diff
@@ -127,7 +127,10 @@ module "bootstrap" {
 
   pxe_kernel             = local.pxe_kernel
   pxe_initrd             = local.pxe_initrd
-  pxe_kernel_args        = local.kernel_args
+  pxe_kernel_args        = concat(
+    local.pxe_kernel_args,
+    ["${var.pxe_kernel_args}"],
+  )
```

Comment:
oof, i hadn't known that pxe_kernel_args was already in use when internally when i chose it as a tfvar